Stripe/StripeiOSTests/STPPaymentMethodFunctionalTest.swift

```diff
@@ -139,8 +139,9 @@ class STPPaymentMethodFunctionalTest: STPNetworkStubbingTestCase {
                                    ephemeralKeySecret: customerAndEphemeralKey.ephemeralKeySecret)
 
         // Element/Sessions endpoint should de-dupe payment methods with CustomerSesssion
-        let cscs = try await STPTestingAPIClient.shared().fetchCustomerAndCustomerSessionClientSecret(customerID: customerAndEphemeralKey.customer,
-                                                                                               merchantCountry: nil)
+        let cscs = try await STPTestingAPIClient.shared().fetchCustomerAndCustomerSessionClientSecret(componentName: "mobile_payment_element",
```

Comment:
Do we need to pass in `componentName`? Can we hardcode it in the `fetch...` method instead? I don't see any other users.